qt/python/mantidqt/widgets/workspacedisplay/matrix/presenter.py

```diff
@@ -84,6 +85,62 @@ def action_copy_bin_values(self, table):
         num_rows = self.model._ws.getNumberHistograms()
         self.copy_bin_values(table, ws_read, num_rows)
 
+    def action_copy_spectrum_to_table(self, table):
+        selected_rows = [i.row() for i in table.selectionModel().selectedRows()]
+        if not selected_rows:
+            self.notify_no_selection_to_copy()
+            return
+        ws = table.model().ws
+        table_ws = CreateEmptyTableWorkspace(OutputWorkspace=ws.getName() + "_spectra")
```

Comment:
I believe we're supposed to use `.name()` instead of `.getName()`
```suggestion
        table_ws = CreateEmptyTableWorkspace(OutputWorkspace=ws.name() + "_spectra")
```